### DISCUSSION TITLE
Documentation on the learning parameters of Uppaal Stratego.

### DISCUSSION THREAD

**Discussion by mael-1234** [UPPAALModelChecker/UPPAAL-Meta#231] [General]
Hello, I have been using Uppaal Stratego for some time now, but I am encountering issues during the optimization phase. I would like to inquire if there is any documentation or article available regarding the learning parameters of Uppaal Stratego.

**Comment by thorulf4** [comment]
These parameters have yet to be documented properly. But I would recommend taking a short look at https://github.com/orgs/UPPAALModelChecker/discussions/227 if you are not already aware of partial-refinement and partial observability.
